suite-common/wallet-graph/src/graphUtils.ts

```diff
@@ -449,31 +450,45 @@ export const calcFakeGraphDataForTimestamps = (
     return sortedData;
 };
 
+let zeroFiatRatesCache: { [currency: string]: number } | undefined;
+
+export const getZeroFiatRates = async ({
+    timestampDatesInTimeFrame,
+}: {
+    timestampDatesInTimeFrame: number[];
+}) => {
+    // we can cache the result because currency symbols should be always same
+    if (zeroFiatRatesCache) return zeroFiatRatesCache;
+
+    // pass arbitrary coin to get supported fiat currency rates - we need this to get currency symbols
+    const anyFiatRates = await getFiatRatesForTimestamps(
+        { symbol: 'btc' },
+        timestampDatesInTimeFrame,
+    );
+
+    const fiatRates = (await anyFiatRates)?.tickers?.[0]?.rates;
```

Comment:
nit: I don't like this wrapping of awaited function too much. I'm kind of used to storing the result into a variable to then use it for better readability.

Reply:
I would probably keep it this way, IMHO it's better like this instead of another name including zeroRates in it, just because I want to use it on next line